frontend/containers/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -24,6 +25,71 @@ import DropdownNative from 'components/DropdownNative/DropdownNative'
 import Peer from 'components/Peer/Peer'
 import Toolbar from 'components/Toolbar/Toolbar'
 
+const FIELD_PUBLISH_SAVE_OPTIONS_LAST_USED = 'publishSaveOptionsLastUsed',
```

Comment:
I know it's less bloated, but can we avoid comma-separated declarations and have one `const`/`let` per variable?

Reply:
Yep done